### PR TITLE
Readme.md enhancements

### PR DESCRIPTION
**Pull request by lykinsbd**
Enforced some more strongly worded "Don't use this in Production" wording.

Also added:

- .dockerignore
- .gitignore

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,37 +1,63 @@
...
 
-> This is **not** for production.
+> This container is **not** for production use!
 
-`Nautobot Lab` is an all-in-one docker container that allows a user to quickly get an instance of Nautobot up and running with minimal effort, for the purposes of kicking the tires. With that said; `nautobot-lab` is **NOT** a ready for production container image.
+`Nautobot Lab` is an all-in-one Docker container that allows a user to quickly get an instance of Nautobot up and running with minimal effort.
+
+This image is for the purposes of "kicking the tires" of Nautobot. Utilize `nautobot-lab` to quickly see if Nautobot is right for you and your organization.
+
+It bears repeating, `nautobot-lab` is **NOT a ready for production container**. If you wish to Nautobot in production via Docker, please see the primary [Nautobot repository](https://github.com/nautobot/nautobot).
```
Comment (glennmatthews) [inline README.md]: Possible change:

```suggestion
It bears repeating, `nautobot-lab` is **NOT a ready for production container**. If you wish to use Nautobot in production, please refer to the [Nautobot documentation](https://nautobot.readthedocs.io/en/latest/installation/).
```

File: README.md [hunk 2]
```diff
@@ -1,37 +1,63 @@
...
-```
+```shell
 docker run -itd --name nautobot -p 8000:8000 nautobot-lab
 ```
 
 ## Creating a Super User
-Once the container has started and all the services have stabalized. A super user account will need to be created to start kicking the tires with what is available in Nautobot. The `createsuperuser` command will prompt you for a username, email address, and password. 
-```
+
+Once the container has started and all the services have stabilized. A super user account will need to be created to start kicking the tires with what is available in Nautobot. The `createsuperuser` command will prompt you for a username, email address, and password.
```
Comment (glennmatthews) [inline README.md]:
```suggestion
Once the container has started and all the services have stabilized, you will need to create a superuser account to start exploring Nautobot. The `nautobot-server createsuperuser` command will prompt you for a username, email address, and password. The email address is unused in this particular workflow and can be left blank.
```

File: README.md [hunk 3]
```diff
@@ -41,5 +67,13 @@ Superuser created successfully.
 ```
 
 ## Kick the Tires
+
 At this point, Nautobot can be accessed at `http://localhost:8000` with the user credentials that were created.
+
+Explore, test, create, destroy, do whatever you like in this lab instance of Nautobot.
+
+If you have any questions, don't hesitate to reach out in the #Nautobot channel on the [Network To Code slack instance](https://networktocode.slack.com), we'll be happy to assist you!
```
Comment (glennmatthews) [inline README.md]:
```suggestion
If you have any questions, don't hesitate to reach out in the #Nautobot channel on the [Network To Code Slack instance](https://networktocode.slack.com), we'll be happy to assist you!
```

**Review by glennmatthews — APPROVED** [review]
A few suggestions, but looks good!

File: README.md [hunk 4]
```diff
@@ -1,37 +1,63 @@
...
-```
+
+Building the container yourself isn't needed to get up and running quickly. The image is hosted on Docker Hub for public consumption, and you can download and start it with a single command. You will still need to [create a Super User as shown below](#Creating-a-Super-User).
+
+```shell
 docker run -itd --name nautobot -p 8000:8000 networktocode/nautobot-lab
 ```
 
+Because this image is an all-in-one container (with Nautobot, PostgreSQL, and Redis), it will take a few seconds to download the container, and then a few seconds more for all of the services to start and stabilize. Once the container has started and all services have stabilized, the web interface can be accessed via `http://localhost:8000`.
```
Comment (jtdub) [inline README.md]: I'm using the default `HEALTCHECK` values on boot up. It takes 30 seconds total to boot and come into a healthy state. Perhaps we should call that out.

**Review by jtdub — CHANGES_REQUESTED** [review]
Thanks for the PR @lykinsbd! I only have one addition, if you don't mind! :)

**Comment by lykinsbd** [comment]
Thank you @glennmatthews and @jtdub addressed those edits. 👍🏻

**Review by jtdub — APPROVED** [review]

**Pull request merged**